src/App.css

```diff
@@ -1,38 +1,288 @@
+    font-family: 'Silkscreen', cursive;
+}
+
+h1 {
+    text-align: center;
+}
+
+a {
+    text-decoration: none;
+    color: black;
+}
+
+h1,
+h3,
+p,
+li,
+label,
+input,
+button {
+    font-family: 'Silkscreen', cursive;
+}
+
+.contentContainer > * {
+    margin: 10px 0;
+    width: 650px;
+}
+
+button {
+    cursor: pointer;
+    background-color: #60a6df;
+    color: white;
+    padding: 10px 20px;
+    text-align: center;
+    border: 1px solid #60a6df;
+    letter-spacing: 0.8px;
+    transition: 0.25s;
+    max-width: 210px;
+}
+
+button:hover,
+button:focus {
+    background-color: #538fc0;
+    color: black;
+}
+
+.disabledButton {
+    cursor: default;
+    background-color: grey;
+    color: white;
+    border: 1px solid grey;
+}
+
+.disabledButton:hover,
+.disabledButton:focus {
+    background-color: grey;
+    color: white;
+}
+
+h1,
+p,
+input,
+fieldset {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    background-image: url('./assets/background/body-background.jpg');
+    background-position: center;
+    background-size: cover;
+    background-repeat: no-repeat;   
+}
+
+.wrapper {
+    max-width: 1280px;
+    margin: 0 auto;
+    width: 90%;
+}
+
+img {
+    max-width: 100px;
+    display: block;
+}
+
+.imageContainer {
+    width: 150px;
+    height: 150px;
+    border-radius: 50%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.woodenSwordContainer,
+.dLongSwordContainer {
+    background-position: center;
+    background-repeat: no-repeat;
+}
+
+.woodenSwordContainer {
+    background-image: url('./assets/swords/stinky-lines.gif');
+    background-size: 210%;
+}
+
+.dLongSwordContainer {
+    background-image: url('./assets/swords/blazing-fire.gif');
+    background-size: 103%;
+}
+
+.username {
+    color: #af0d91;
+    font-weight: bold;
+}
+
+.woodenSword {
+    color: #0c7222;
+    font-weight: bold;
+}
+
+.dragonLongSword {
+    color: #800000;
+    font-weight: bold;
+}
+
+.displayContainer {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    height: 100vh;
+}
+.contentContainer {
+    border: 3px solid black;
+    min-height: 450px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-color: rgba(255, 255, 255, 0.555);
+    text-align: center;
+    overflow-wrap: break-word;
+}
+
+.userStatsContainer {
+    overflow-wrap: break-word;
+    height: auto;
+}
+
+.landingPageForm {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.landingPageForm > * {
+    margin: 30px 0;
+}
+
+.adventureContainer {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+}
+
+.logContainer {
+    width: calc((100% / 3) - 20px);
+    margin: 10px;
+    padding: 5px;
+    border: 3px solid #d6a241;
+    background-image: url('./assets/background/parchment-paper-background.jpg');
+}
+
+.adventureTask {
+    margin: 10px 0;
+}
+
+.displayContainer.ChallengesS2Container {
```

Comment:
The container did not fit the whole screen, there was white space underneath the footer while on this page

Reply:
hid footer overflow :heavy_check_mark: